### ISSUE TITLE
Warning from the characteristic Active power since update to version 7

### ISSUE THREAD

**Issue by xorguy** [grzegorz914/homebridge-enphase-envoy#173]
Hi.

Since Enphase updated my Envoy from 5 to D7.0.88 I changed the settings at the plugin but now I get the following warnings. 

```shell
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Consumption Net power peak reset: Off
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV, active power: 1.816536 kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV L1, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV L2, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV L3, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV, apparent power: 1.827546 kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV L1, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV L2, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data PV L3, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid, active power: -1.00736 kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid L1, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid L2, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid L3, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid, apparent power: -1.128463 kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid L1, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid L2, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Grid L3, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load, active power: 0.809176 kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load L1, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load L2, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load L3, active power: NaN kW
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load, apparent power: 0.699083 kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load L1, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load L2, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] Device: 10.0.0.113 Envoy-S, Live Data Load L3, apparent power: NaN kVA
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L1': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L2': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L3': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L1': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L2': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L3': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L1': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L2': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L3': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L1': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L2': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L3': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L1': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L2': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Active power L3': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L1': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L2': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
[12/16/2024, 12:24:22 PM] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Apparent power L3': characteristic value expected valid finite number and received "NaN" (number). See https://homebridge.io/w/JtMGR for more info.
``` 
Any way to remove/fix those?

Regards.

**Comment by grzegorz914** [comment]
Emanować debug in plugin config and then post the log

**Comment by xorguy** [comment]
Hi, attached a logfile, let me know if it is enough.

[h4.txt](https://github.com/user-attachments/files/18157605/h4.txt)

**Comment by grzegorz914** [comment]
Ok, will fix this issue in the next update.

**Comment by grzegorz914** [comment]
fixed in v9.6.9
